helpers/setup.sh

```diff
@@ -1,15 +1,55 @@
 #!/bin/bash
 
+# Text color variables
 PURPLE_BOLD=$'\e[1;95m'
 PURPLE_REGULAR=$'\e[0;95m'
 RESET_TEXT=$'\e[0;0m'
 
+
+# Installation
+function install () {
+    echo
+	echo "${PURPLE_REGULAR}Installing dependencies . . .${RESET_TEXT}"
+	echo
+	poetry install
+	echo "${PURPLE_REGULAR}Installation complete. Use ${PURPLE_BOLD}'poetry shell'${PURPLE_REGULAR} to activate a virtual environment.${RESET_TEXT}"
+	echo
+}
+
+function aws_cloudshell_install () {
+    echo
+    echo "${PURPLE_REGULAR}Installing dependencies for AWS CloudShell. See repo ${PURPLE_BOLD}README ${PURPLE_REGULAR}for more details."
+    echo
+    curl -sSL https://install.python-poetry.org | python3 - # Install Poetry
+    sudo yum install -y yum-utils # Install yum-utils so Terraform can be installed
+    sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo 
+    sudo yum -y install terraform
+    sudo amazon-linux-extras install epel # Install extras for things like tab completion, etc
+    sudo yum install bash-completion-extras
+    echo
+    echo "${PURPLE_REGULAR}Installation complete.${RESET_TEXT}"
+    echo
+}
+
+
+# Demo infrastructure
+function demo_infra_provision () {
+    echo
+    echo "${PURPLE_REGULAR}Running terraform apply. See repo ${PURPLE_BOLD}README ${PURPLE_REGULAR}for more details.${RESET_TEXT}"
+    echo
```

Comment:
While unlikely, adding a variable guard will help prevent invocation or misconfiguration from mucking up things. For example

```suggestion
function demo_infra_provision () {
    PROJECT="$1"
    if [ -z "$PROJECT" ]; then
        echo "Must specify 1 argument as the directory in resources/example-policies-infrastructure/"
        exit 1
    fi
    
    if [ ! -d resources/example-policies-infrastructure/$PROJECT ]; then
        echo "$PROJECT is not a valid example policy infrastructure"
        exit 1
    fi

    echo
    echo "${PURPLE_REGULAR}Running terraform apply. See repo ${PURPLE_BOLD}README ${PURPLE_REGULAR}for more details.${RESET_TEXT}"
    echo
```